winui/Shimmer/Built-in-Types.md

```diff
@@ -9,11 +9,11 @@ documentation: ug
 
 # Built-in Types of WinUI Shimmer Control (SfShimmer)
 
-The Syncfusion WinUI Shimmer (SfShimmer) control offers a range of built-in shimmer types, each designed to serve specific user interface needs. These built-in types provide convenient options for creating captivating loading animations in your WinUI applications. The following built-in shimmer types such as CirclePersona, SquarePersona, Profile, Article, Video, Feed, and Shopping is available in Shimmer control.
+The Syncfusion [WinUI Shimmer control](https://www.syncfusion.com/winui-controls/shimmer) (SfShimmer) offers a range of built-in shimmer types, each designed to serve specific user interface needs. These built-in types provide convenient options for creating captivating loading animations in your WinUI applications. The following built-in shimmer types such as [CirclePersona](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Core.ShimmerType.html#Syncfusion_UI_Xaml_Core_ShimmerType_CirclePersona), [SquarePersona](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Core.ShimmerType.html#Syncfusion_UI_Xaml_Core_ShimmerType_SquarePersona), [Profile](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Core.ShimmerType.html#Syncfusion_UI_Xaml_Core_ShimmerType_Profile), [Article](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Core.ShimmerType.html#Syncfusion_UI_Xaml_Core_ShimmerType_Article), [Video](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Core.ShimmerType.html#Syncfusion_UI_Xaml_Core_ShimmerType_Video), [Feed](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Core.ShimmerType.html#Syncfusion_UI_Xaml_Core_ShimmerType_Feed), and [Shopping](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Core.ShimmerType.html#Syncfusion_UI_Xaml_Core_ShimmerType_Shopping) is available in Shimmer control.
```

Comment:
@GilbertmanojSF3971 The WinUI Shimmer control offers a range

Reply:
@Manivannan-E  Sir. Modified the content as per your suggestion and removed the syncfusion text before the control name.